config.json

```diff
@@ -243,6 +243,18 @@
       "topics": [
         "Result"
       ]
+    },
+	{
```

Comment:
> I would suggest that the indentation here use spaces instead of tabs, otherwise formatters such as https://github.com/exercism/rust/blob/master/bin/init_exercise.py will cause this line to change for a potentially unrelated change.

same